litgpt/tokenizer.py

```diff
@@ -54,6 +54,14 @@ def __init__(self, checkpoint_dir: Union[Path, str]) -> None:
         else:
             raise NotImplementedError
 
+        # NOTE: A temporary fix until it's resolved on Tokenizers side.
+        # LlaMA tokenizer strips leading spaces if to decode a single token at a time.
+        # https://github.com/huggingface/transformers/issues/31643
+        self.apply_decoding_fix = None
+        if (config_path := checkpoint_dir / "tokenizer_config.json").is_file():
```

Comment:
Finally the walrus operator in action 😅

Reply:
I'm not the one who started this. Even in this file there are many other occurrences of this operator 😊